ios/Gekidou/Sources/Gekidou/Storage/Database+Category.swift

```diff
@@ -4,9 +4,9 @@ import SQLite
 extension Database {
     public func queryCategoryId(inTeamId teamId: String, type: String, forServerUrl serverUrl: String) -> String? {
         if let db = try? getDatabaseForServer(serverUrl) {
-            let idCol = Expression<String>("id")
-            let teamIdCol = Expression<String>("team_id")
-            let typeCol = Expression<String>("type")
+            let idCol = SQLite.Expression<String>("id")
```

Comment:
Are these changes because of Xcode 16? If they are, it could’ve been easier to handle those in a separate PR beforehand. It would help keep the focus on the main bug fix here.

Reply:
I admit, when I saw 23 files changed, I made a note to myself that I need to carve a bit more time to run through this PR.